gammapy/catalog/hawc.py

```diff
@@ -0,0 +1,126 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+HAWC catalogs (https://www.hawc-observatory.org)
+"""
+
+from .core import SourceCatalog, SourceCatalogObject
+from ..utils.scripts import make_path
+from astropy.table import Table
+import numpy as np
+
+class SourceCatalogObject2HWC(SourceCatalogObject):
+    """One source from the HAWC 2FHL catalog.
+
+    Catalog is represented by `~gammapy.catalog.SourceCatalog2HWC`.
+    """
+
+    _source_name_key = 'source_name'
+    _source_index_key = 'catalog_row_index'
+
+    def info(self, info='all'):
+        """Print info.
+
+        Parameters
+        ----------
+        info : {'all', 'basic', 'position', ' spectra'}
+            Comma separated list of options
+        """
+        ss = self.__str__(info=info)
+        print(ss)
+
+    def __str__(self, info='all'):
+        """Info string.
+
+        Parameters
+        ----------
+        info : {'all', 'basic', 'position', 'spectra} #TODO: call spectra different
```

Comment:
Is the word 'spectra' as argument ok?

Reply:
I see that HGPS has "spec" and Fermi has "spectral".
I would suggest "spectrum" everywhere.
If you can make it consistent here, please go a head.
If no, just leave as-is and I'll do some cleanup in a follow-up commit when merging this PR.
(I almost always do this nowadays to avoid going back and forth on details when I could just as well make the edits directly when I review the code of the PR after opening it up in PyCharm.)

Reply:
Changed it (in hess.py and fermi.py as well)